# Probabilidad y Estadística
## Facilitador: José Gabriel Rodríguez Rivas
## Alumna: Lluvia A. Saenz S.

La distribución de **Poisson** se utiliza cuando estamos interesados en el número de eventos que ocurren en un intervalo fijo de tiempo o espacio. Los eventos ocurren de manera independiente y a una tasa constante.

En cambio la dsitribucion **binomial** se utiliza cuando estamos interesados en el número de éxitos en un número fijo de ensayos independientes.
Cada ensayo tiene dos posibles resultados: éxito o fracaso.


In [18]:
from scipy.stats import poisson
from scipy.stats import binom

---
**EJERCIRCIO 1**
---


El número x de personas ingresadas a una unidad de cuidados intensivos en un hospital particular, en un día, tiene una distribución de **probabilidad de Poisson** con media igual a cinco personas por día.

• a) ¿Cuál es la probabilidad de que el número de personas ingresadas a una unidad de cuidados intensivos en un hospital particular, en un día particular, sea dos?

• b) ¿Es probable que x exceda de 10? Explicar

In [14]:
# media personas por dia
x = 5
# Probabilidad de que el número de personas ingresadas sea 2
k = 2
probab_kx_2 = poisson.pmf(k,x)
print("Probabilidad de que el número de personas ingresadas sea 2:", probab_kx_2)


Probabilidad de que el número de personas ingresadas sea 2: 0.08422433748856832


In [15]:
probab_mayor_10 = 1 - poisson.cdf(10, x)
print("Probabilidad de que el número de personas ingresadas exceda de 10:", probab_mayor_10)

Probabilidad de que el número de personas ingresadas exceda de 10: 0.013695268598382881


a) La probabilidad de que exactamente dos personas ingresen en un día es aproximadamente 0.0842, lo cual es extremadamente baja.

b) La probabilidad de que el número de personas ingresadas exceda de 10 es aproximadamente 0.0134, es bastante improbable que más de 10 personas ingresen en un solo día.

---
**EJERCIRCIO 2**
---

 Un fabricante de podadoras para el pasto compra motores de 1 hp y 2 ciclos, en lotes de 1,000, a un proveedor.

El fabricate equipa cada una de las podadoras producidas por su planta con uno de los motores.

La historia muestra que la probabilidad de que cualquier motor del proveedor resulte no satisfactorio es .001.

En un embarque de 1,000 motores, Calcular

• ¿cuál es la probabilidad de que ninguno sea defectuoso?

• ¿cuál es la probabilidad de que tres sean defectuosos?

• ¿cuál es la probabilidad de que cuatro sean defectuoso?


`El fabricante de podadoras compra motores en lotes de 1,000, y la probabilidad de que cualquier motor resulte no satisfactorio (defectuoso) es 0.001. cada motor puede tener dos resultados posibles (defectuoso o no defectuoso), y hay un número fijo de ensayos (1,000). Por lo tanto, este es un problema de distribución binomial.`

In [24]:
prob_fail = 0.001
lotes = 1000

# Probabilidad de 0 defectuosos
x_0=0
probab_x_0 = binom.pmf(x_0, lotes, prob_fail)
print("Probabilidad de que ninguno sea defectuoso:", probab_x_0)
# Probabilidad de 3 defectuosos
x_3=3
probab_x_3 = binom.pmf(x_3, lotes, prob_fail)
print("Probabilidad de que ninguno sea defectuoso:", probab_x_3)
# Probabilidad de 4 defectuosos
x_4=4
probab_x_4 = binom.pmf(x_4, lotes, prob_fail)
print("Probabilidad de que ninguno sea defectuoso:", probab_x_4)


Probabilidad de que ninguno sea defectuoso: 0.3676954247709639
Probabilidad de que ninguno sea defectuoso: 0.061282509389840666
Probabilidad de que ninguno sea defectuoso: 0.015289955420838651


36.76% de que niguno sea defectuoso

6.12% de 3 sean defectuoso

1.52% de 4 sean defectuoso